strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -64,38 +61,28 @@ public void init(TestInfo testInfo)
         super.init(testInfo);
     }
 
-    private Set<MutableRepository> getRepositories(TestInfo testInfo)
-    {
-        Set<MutableRepository> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("rmmcjtit-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("trmir-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-
-        return repositories;
-    }
-
-    @AfterEach
-    public void removeRepositories(TestInfo testInfo)
-            throws Exception
-    {
-        removeRepositories(getRepositories(testInfo));
-    }
-
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRebuildArtifactsMetadata(TestInfo testInfo)
+    public void testRebuildArtifactsMetadata(@MavenRepository(storageId = STORAGE0,
+    														  repositoryId = RMMCJTIT_SNAPSHOTS)
+                                                              Repository repository,
+                                             @MavenTestArtifact(repositoryId = RMMCJTIT_SNAPSHOTS,
+                                                                id = "org.carlspring.strongbox:strongbox-metadata-one",
+                                                                versions = {"2.0-20190512.202015-1",
+    												 			   	        "2.0-20190512.202015-2",
+    												 			   	        "2.0-20190512.202015-3",
+    												 			   	        "2.0-20190512.202015-4",
+    												 			   	        "2.0-20190512.202015-5"},
+                                                                classifiers = {"javadoc",
+                                                                               "sources",
+    				                                                      	   "source-release"})
+                                                                List<Path> artifact)
             throws Exception
```

Comment:
Cosmetic: Could you please follow the project's coding convention? The code above should look like this:

```
    public void testRebuildArtifactsMetadata(@MavenRepository(storageId = STORAGE0,
    								                             						  repositoryId = RMMCJTIT_SNAPSHOTS)
                                             Repository repository,
                                             @MavenTestArtifact(repositoryId = RMMCJTIT_SNAPSHOTS,
                                                               id = "org.carlspring.strongbox:strongbox-metadata-one",
                                                               versions = { "2.0-20190512.202015-1",
    												 			   	                                      "2.0-20190512.202015-2",
    												 			   	                                      "2.0-20190512.202015-3",
    												 			   	                                      "2.0-20190512.202015-4",
    												 			   	                                      "2.0-20190512.202015-5" },
                                                               classifiers = { "javadoc",
                                                                               "sources",
    				                                                        	         "source-release" })
                                             List<Path> artifact)
            throws Exception
```

Reply:
![image](https://user-images.githubusercontent.com/18516640/60390018-2dc4ee80-9a9b-11e9-9070-068ca6723eca.png)

@carlspring 
Not sure if github changed the spacing in the coding that you pasted. But, I have attached the screenshot of the changes. Is this how it should look?

Reply:
You seem to be using tabs and spaces. Please, copy the code from my example. We don't use tabs, we use 4 spaces. Please, have a look at [our coding convention](https://strongbox.github.io/developer-guide/coding-convention.html) and try to follow it. Thanks! :)

Reply:
This is done. Thanks!

Reply:
If it's done, I don't think you've pushed it, as the diff of your pull request is unchaged.

Reply:
Can you please check now?

Reply:
Thanks! It looks okay now! :)